site/content/docs/main/migration-case.md

```diff
@@ -50,5 +50,9 @@ Check that the second cluster is behaving as expected:
 
 If you encounter issues, make sure that Velero is running in the same namespace in both clusters.
 
+## Migrating Workloads Across Different Kubernetes Versions
+
+Migration across clusters that are not running the same version of Kubernetes is possible with Velero. For more information, look at the [EnableAPIGroupVersions](enable-api-group-versions-feature.md) feature.
```

Comment:
Migrating between kubernetes versions doesn't strictly require this feature flag, so long as there were no incompatible changes to the core types. This is a little tricky to provide guidance on, because while I know v1.15.->1.16 did this, not all versions do, and I don't really want to maintain an exhaustive list of version jumps that this is relevant to. 

Could we maybe add some wording to this about Kubernetes upgrades containing incompatible or removed API groups?

Reply:
We'll address these comments in a different issue or PR.

Reply:
Thanks, @nrb! I have some additions per your comment, but I'm going to consult my go-to Kubernetes expert, @brito-rafa, to see if I've covered what was intended. Hopefully, I'll have the commit up by end of today 😃

Reply:
@nrb, I have updated the paragraph. Let me know if that doesn't work.